README.md

```diff
@@ -89,7 +103,9 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An ERD (Entity Relationship Diagram) is a visual representation of the way separate entities within a database relate. The ERD can map many important pieces of information, including -
+1) Which data relates to other data (for example above, the relationship lines will likely form between Genies, and Lamps, and probably Lamps, and Perons[people], BUT there will be no lines between Genies and Pets, because there is no relationship between the two entities)
+2) What type of relationship the connected entities have. One-to-many, in the case of one Genie/Lamp having many persons throughout their lifecycle, and each person belonging to only one Genie/Lamp. Or many-to-many, in the case of many persons (say in a family of five) having many pets, while also belonging to many pets (say the family has three dogs, four cats, and a rat) which have many owners.
 ```
```

Comment:
:tada: 
I bet the rat rules the house.  In my house, it's the cats.  They even tell the big dog what to do.